.idea/modules/delivery-sdk-java.iml

```diff
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<module external.linked.project.id="delivery-sdk-java" external.linked.project.path="$MODULE_DIR$/../.." external.root.project.path="$MODULE_DIR$/../.." external.system.id="GRADLE" external.system.module.group="com.kenticocloud" external.system.module.version="1.0.1-SNAPSHOT" type="JAVA_MODULE" version="4">
```

Comment:
These iml files will probably conflict with master... but you can check them in as is and I can resolve them.

Reply:
When pushing the commit, I wasn't really sure about why the "1.0.1-SNAPSHOT" was changed to "1.0.1" automatically. I've concluded that you've somehow marked the 1.0.1 version of the Maven entry as production-grade. Is there anything I could do better next time? (Sorry, I haven't had any experience with Java before.)

Reply:
No reason to be sorry, I have seasoned Java devs who work for me who get confused by these intellij project files, and frankly, there are a lot of intellij plugin developers who write to these incorrectly as well.

IntelliJ maintains a separate build system than gradle, which is annoying, but their build system is very fast.  As such, it needs to be kept in sync with the gradle build.  Intellij will try to detect changes to the gradle files and pop up a dialog asking you to import them, and then it updates the .iml files, which can be checked in.  This can be done manually as well by going to the Gradle tool window (easiest way to get to any tool window or recently used file is ctrl+e).

Regardless, I had already changed the master branch to 1.0.2 at that point, so there would have been a conflict anyways.

Sometime when things settle down, I should setup a chat with you and/or Petr and give a quick tutorial of things if you guys are interested... at the very least pass on knowledge of the CI build.

Reply:
Great, thanks! BTW, I like the Ctrl+E shortcut.

Reply:
Sure we are  interested. I have it in my TODO list as well :) Just waiting till things get less busy...